registry/templates/nav.html

```diff
@@ -6,8 +6,14 @@
 
   <ul class="navbar-nav mr-auto">
     {% if current_user and current_user.is_authenticated %}
-    <li class="nav-item">
-      <a class="nav-link" href="{{ url_for('donor.overview') }}">Přehled dárců</a>
+    <li class="nav-item dropdown">
+      <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
+        Dárci
+      </a>
+      <div class="dropdown-menu">
+          <a class="dropdown-item" href="{{ url_for('donor.overview') }}">Přehled dárců</a>
```

Comment:
Menu bych udělal tak, aby důležité položky byly dostupné přímo:
- Přehled dárců
- Import dat
- Přehledy
  - Přehled importů
  - Ignorovaní dárci

Reply:
Vyřešeno v dalším commitu